.gitmodules

```diff
@@ -25,3 +25,6 @@
 [submodule "RTTR/languages"]
 	url = https://github.com/Return-To-The-Roots/languages.git
 	path = RTTR/languages
+[submodule "contrib/kaguya"]
+	path = contrib/kaguya
+	url = https://github.com/satoren/kaguya.git
```

Comment:
We don't really depend on it (in a bad way). This is what submodules are for: Pull in some *specific* source code. So we just pull in another library at a version we choose. Why should we have extra work for maintaining our copy of that repo?